### PR TITLE
Feat: Avatar Badge On Live

### PR DESCRIPTION
**Pull request by Rishi05051997**
Please review avatar badge code

### CODE REVIEW

File: components/Avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+  <body>
+    <div class="container">
+      <!-- Header starting here -->
+      <div class="header_container">
+        <header class="sticky">
+          <img
+            class="img_logo"
+            src="./../../assets/image section/logo.png"
+            height="50"
+            width="50"
```
Comment (Mohasin-Haque) [inline components/Avatar/avatar.html]: avoid inline styling

File: components/Avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+          </div>
+          <div class="basic_container pad-md mar-y-2">
+            <iframe
+              src="https://carbon.now.sh/embed/t0dPjNUCMqBq3v6nkovf"
+              style="
+                width: 437px;
+                height: 467px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
```
Comment (Mohasin-Haque) [inline components/Avatar/avatar.html]: this can also be done in css file

File: styles/badge.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        margin: var(--mar-xxl) 0;
+    }
+
+>>>>>>> 6bcad2a2134d586ae9b9ae453d3997124bed1d61
+    /* Badges Starting Here */
+    .badge_cont {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-around;
+<<<<<<< HEAD
```
Comment (Mohasin-Haque) [inline styles/badge.css]: whats this??

File: styles/badge.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        top: 0.1rem;
+        color: #fff;
+        font-weight:700;
+<<<<<<< HEAD
+    }
+=======
+
+      }
+>>>>>>> 6bcad2a2134d586ae9b9ae453d3997124bed1d61
+
```
Comment (Mohasin-Haque) [inline styles/badge.css]: is it conflict showing ir what?

**Review by Mohasin-Haque — COMMENTED** [review]
Your code looks good. but please go through comments once

File: components/Avatar/avatar.html (hunk 2 above)
Comment (Rishi05051997) [reply components/Avatar/avatar.html]: no that file is code file carbon.sh iframe

File: styles/badge.css (hunk 3 above)
Comment (Rishi05051997) [reply styles/badge.css]: I resolved that it's conflict

File: styles/badge.css (hunk 4 above)
Comment (Rishi05051997) [reply styles/badge.css]: Yes that's conflict

File: components/Avatar/avatar.html [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+              class="md-height xl-bdr-radius"
+              src="./../../assets/avatar.png"
+              height="50"
+              alt=""
+            />
+            <img
+              class="xl-height xl-bdr-radius"
+              src="./../../assets/avatar.png"
+              height="50"
+              alt=""
```
Comment (darshandamre) [inline components/Avatar/avatar.html]: put some alt text here

File: components/Badge/bagde.html [hunk 6]
```diff
@@ -0,0 +1,336 @@
...
+                alt=""
+                class="md-height xl-bdr-radius"
+              />
+              <div class="onlineBadge"></div>
+            </div>
+            <div class="badge_div">
+              <img
+                src="./../../assets/avatar.png"
+                alt=""
+                class="md-height xl-bdr-radius"
```
Comment (darshandamre) [inline components/Badge/bagde.html:263]: same here, add some alt text

File: components/Badge/bagde.html [hunk 7]
```diff
@@ -0,0 +1,336 @@
...
+              <div class="notifyBadge"><span>20</span></div>
+            </div>
+            <div class="badge_div">
+              <span class="iconify" data-icon="icon-park-outline:like"></span>
+              <div class="notifyBadge"><span>20</span></div>
+            </div>
+            <div class="badge_div">
+              <img
+                src="./../../assets/avatar.png"
+                alt=""
```
Comment (darshandamre) [inline components/Badge/bagde.html:313]: same here

File: styles/badge.css [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+    
+
+    
+
+    
+
+  
+
+
```
Comment (darshandamre) [inline styles/badge.css:81]: use a formatter, get rid of this empty space

**Review by darshandamre — COMMENTED** [review]
i think rest of the code is fine

File: styles/badge.css (hunk 8 above)
Comment (Rishi05051997) [reply styles/badge.css:81]: Thank You I removed that.

File: components/Avatar/avatar.html (hunk 5 above)
Comment (Rishi05051997) [reply components/Avatar/avatar.html]: Ok Fine adding now

File: components/Badge/bagde.html (hunk 6 above)
Comment (Rishi05051997) [reply components/Badge/bagde.html:263]: Adding

File: components/Badge/bagde.html (hunk 7 above)
Comment (Rishi05051997) [reply components/Badge/bagde.html:313]: Adding

**Pull request merged**